lib/browser/uriToBlob.js

```diff
@@ -3,18 +3,11 @@
  * React Native to retrieve a file (identified by a file://
  * URI) as a blob.
  */
-export default function uriToBlob (uri) {
-  return new Promise((resolve, reject) => {
-    const xhr = new XMLHttpRequest()
-    xhr.responseType = 'blob'
-    xhr.onload = () => {
-      const blob = xhr.response
-      resolve(blob)
-    }
-    xhr.onerror = (err) => {
-      reject(err)
-    }
-    xhr.open('GET', uri)
-    xhr.send()
-  })
+export default async function uriToBlob (uri) {
+  try {
+    const res = await fetch(uri)
```

Comment:
Can you confirm (maybe using some documentation) that using the Fetch API is equivalent to XHR for React Native ? We do not have automated tests for React Native, so we have to be very careful here to not introduce any bugs.

Reply:
I can say, from [this link](https://reactnative.dev/docs/network#using-fetch), that it is the same to use Fetch API instead of XHR for react native.